version.lua

```diff
@@ -1,5 +1,5 @@
 return{
     ["apkCode"]=62,
-    ["code"]=000611,
-    ["string"]="V0.6.1",
+    ["code"]=000612,
+    ["string"]="V0.6.1-MOD",
```

Comment:
You should not change this, especially when you are making a PR, this will be merged and may mess with Releases tab if MrZ forget to change back